client/src/modules/chapters/pages/chaptersPage.tsx

```diff
@@ -1,20 +1,38 @@
-import { Heading, Stack } from '@chakra-ui/layout';
+import { Flex, Heading, Stack } from '@chakra-ui/layout';
 import { NextPage } from 'next';
 import React from 'react';
 import { Grid, GridItem } from '@chakra-ui/react';
+import { LinkButton } from 'chakra-next-link';
+import { ChapterCard } from '../../../components/ChapterCard';
+import { useChaptersQuery } from '../../../generated/graphql';
+import { useAuth } from '../../../modules/auth/store';
+import { useCheckPermission } from '../../../hooks/useCheckPermission';
+import { Permission } from '../../../../../common/permissions';
 import { Loading } from 'components/Loading';
-import { ChapterCard } from 'components/ChapterCard';
-import { useChaptersQuery } from 'generated/graphql';
 
 export const ChaptersPage: NextPage = () => {
   const { loading, error, data } = useChaptersQuery();
-
+  const { user } = useAuth();
   const isLoading = loading || !data;
   if (isLoading || error) return <Loading loading={isLoading} error={error} />;
 
+  const canAuthenticateWithGoogle = useCheckPermission(
+    Permission.GoogleAuthenticate,
+  );
```

Comment:
Similarly to eventsPage, this has to be called before the conditional.  Otherwise, if it returns early (with the `Loading` component) this hook won't be reached.